### ISSUE TITLE
Document type tag annotations

### ISSUE THREAD

**Issue by DanielSank** [labrad/pylabrad#103]
There is some magic available when writing type tags involving `{}` and other undocumented things. We should document this. It would probably take someone who knows how it works like ten minutes to do this.

See [here](https://github.com/martinisgroup/pylabrad/wiki/Type-dac-annotations).

If someone can explain it to me I'm happy to write the wiki page. I've seen this in the code
```python
@setting(1234, "My Setting", arg=['s{name}', 'i{id}'])
...
```
From the perspective of the network this seems to behave equivalently to
```python
@setting(1234, "My Setting", arg=['s', 'i'])
...
```
so I gather the stuff in `{}` is just source code documentation (which is weird because you can write a doc string anyway). Is there a way to make these annotations visible over the network?

When I load up python and do this
```python
cxn = labrad.connect()
cxn.my_server.do_a_thing
```
I get a doc string. What elements go into generating it? It seems there are at least some parts which are generated automatically.

**Comment by chrismacklin** [comment]
I've seen this as well: `['*v: add one row of data', '*2v: add multiple rows of data']`

**Comment by maffoo** [comment]
Added some discussion of how this works in the [wiki](https://github.com/labrad/pylabrad/wiki/Writing-Servers#setting-decorator).

TL;DR: don't try to get clever with comments in the type tags in the setting decorator. Just put the basic tags and then describe things in words in the doc string.

(I realize that existing code does not always follow this advice, but I think it's the best thing for new servers and we should go fix existing code. I think comments in type tags was a bad idea, and this is an area that we should clean up and simplify in labrad 2...)
